internal/testutil/acc/config_formatter.go

```diff
@@ -50,3 +57,184 @@ func sortStringMapKeys(m map[string]string) []string {
 	sort.Strings(keys)
 	return keys
 }
+func sortStringMapKeysAny(m map[string]any) []string {
+	keys := make([]string, 0, len(m))
```

Comment:
knit: alternatively we could use copy as in: https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/internal/testutil/acc/attribute_checks.go#L108

but it's fine to leave it as it is